native_client/BUILD

```diff
@@ -1,30 +1,62 @@
 # Description: Deepspeech native client library.
 
-cc_library(
-    name = "deepspeech",
-    srcs = ["deepspeech.cc"],
-    hdrs = ["deepspeech.h"],
-    deps = ["//tensorflow/core:core",
-            ":c_speech_features"]
-)
+cc_binary(
+    name = "libdeepspeech.so",
+    srcs = ["deepspeech.cc", "deepspeech.h"],
+    deps = [":c_speech_features",
+            "//tensorflow/core:core_cpu",
```

Comment:
I think for running inference, yes, pretty much - Or at least, we should special case things and perhaps have a separate target for with GPU support if we really want it. Given the limited set of GPUs supported and the intended use-case for the library, I'd question how useful the GPU support is.